### DISCUSSION TITLE
High number of tied hits in diamond output

### DISCUSSION THREAD

**Discussion by hkeward** [bbuchfink/diamond#530] [Q&A] [answered]
Hello `diamond` authors - we have been running `diamond` on Illumina paired-end data (150 bp reads), with the following command line:

```bash
diamond \
    blastx \
    -p ${cores} \
    -b 5 \
    -c 1 \
    -e 0.00001 \
    -k 5 \
    --id 80 \
    -q "${fastq}" \
    -d "${diamond_db}" \
    --outfmt 6 qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore staxids sscinames stitle
```
The diamond database we're using is made from the nr.gz databse found [here](https://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz).

We run this command once for each paired fastq, then pass the output on to a script that determines the top-scoring alignment for each read pair.

We have noticed that for a large proportion of reads, the top alignments may be exactly tied across a number of different proteins/species (`pident`, `length`, `evalue`, and `bitscore` are all identical).

We are wondering if there are any additional output columns you think might be helpful to further distinguish between alignments, or if any of the input parameters could be tweaked to result in more reads that have non-tied alignments.

**Comment by bbuchfink** [comment, marked as answer]
Hi, sorry but if the alignments are identical I don't think changing any diamond parameters can help you here.

**Reply by hkeward** [reply]
Thanks @bbuchfink, appreciate the feedback - I figured, but thought it might be worth a shot!
